apm-sniffer/optional-plugins/trace-ignore-plugin/src/main/java/org/apache/skywalking/apm/plugin/trace/ignore/conf/IgnoreConfigInitializer.java

```diff
@@ -57,6 +60,18 @@ public static void initialize() throws ConfigNotFoundException, AgentPackageNotF
             configFileStream = loadConfigFromAgentFolder();
             Properties properties = new Properties();
             properties.load(configFileStream);
+            PropertyPlaceholderHelper helper =
```

Comment:
@JaredTan95 Is this right usage of your util API? Could you help on review this PR?

Reply:
The usage is right. BTW, @a198720 you can add some test case in `org.apache.skywalking.apm.plugin.trace.ignore.TraceIgnoreTest`? Take `org.apache.skywalking.apm.agent.core.conf.SnifferConfigInitializerTest#testConfigOverridingFromSystemEnv` as a example. :D

Reply:
OK. If you are going to add test. I will wait until you done. Thanks for the contributions.